src/components/TutorialPainter.jsx

```diff
@@ -0,0 +1,282 @@
+import { OrbitControls } from "@react-three/drei";
+import { useFrame, useThree } from "@react-three/fiber";
+import React, { useEffect, useRef, useState } from "react";
+import * as THREE from "three";
+import useChuckStore from "../store/chuckStore";
+import { makeCustomAxis, updateChuckData } from "../utils/chuckUtils";
+import { clickedButton, selectedIndex } from "../utils/makeDogTutorial";
+import Chuck from "./Chuck";
+import ReverseChuck from "./ReverseChuck";
+
+const TutorialPainter = ({
+  rotationAngle,
+  orderIndex,
```

Comment:
```suggestion
  essentialRotateIndex,
```
 위에 orderButton 변수명 변경을 요청 드렸던 것처럼 order을 사용하는 변수명도 이렇게 바꾸면 더 직관적일 것 같아서 코멘트 남깁니다!

Reply:
확인했습니다! 다만 제가 생각했을 땐 orderIndex와 orderButton이 클릭하는 요소만 다르고 같은 역할이라고 생각해서 변수명을 좀 통일시켰으면 합니다. 이에 대해 어떻게 생각하시나요?

Reply:
좋은 의견입니다. 연계되어 있는 로직이다보니 그렇게 하는게 좋을 것 같습니다!

Reply:
변경사항 반영했습니다! 확인 부탁드려요!